File: radar-api/src/main/java/com/gatto/radar/recommendation/RadarItem.java
package com.gatto.radar.recommendation;

import java.time.Instant;
import java.util.List;

public record RadarItem(
        String zoneCode,
        String zoneName,
        Instant capturedAt,

        double score,

        double distanceKm,
        int travelMinutes,

        double demand,
        int pickupMinutes,
        double driverSupply,

        List<String> reasons
) {
}

File: radar-mobile/src/model/RadarItem.ts
export type RadarItem = {
  zoneCode: string;
  zoneName: string;
  capturedAt: string;
  score: number;
  distanceKm: number;
  travelMinutes: number;
  demand: number;
  pickupMinutes: number;
  driverSupply: number;
  reasons: string[];
};
